### ISSUE TITLE
Request:  Add function to bulk "unstack" photos

### ISSUE THREAD

**Issue by macr0dev** [Blendan1/PhotoPrismBulkEditor#1]
Would be great to have the ability to select stacks of photos and have them "unstacked."  
  
Thanks!

**Comment by Blendan1** [comment]
Congratulations 🎉🎉

YOU have opened the first issue on this project


You have won: the warm fuzzy feeling of never being alone 😊

**Comment by macr0dev** [comment]
No errors, but no success either.  It steps through the selected images but doesn't appear to actually click the edit button.

**Comment by Blendan1** [comment]
I commited a fix, seems like if the firs tab wasnt opend before the buttns won't be rendert. this should fix the issue

**Comment by macr0dev** [comment]
Didn't work for me again, but I now understand why.  It's a use case variance.

It doesn't seem to be clicking the unstack button as an independent option.  I did a test where I added some keywords for some photos that were from the same place, and then it worked.  It might have been working the first time if I'd realized it needed that.  Any way to let it trigger without having to edit the metadata as well?

It looks like it accepts "blank" keywords.  So I can definitely make it work as is.  It just may be less intuitive if people want to unstack pictures without also making metadata changes.
